frontend/src/Components/LoginForm/LoginForm.css

```diff
@@ -0,0 +1,85 @@
+.Starter {
```

Comment:
nit: generally in css we use lowercase only in CSS class names

Reply:
Just changed!